src/app/dashboard/components/sidebar/FilterDatePicker.jsx

```diff
@@ -0,0 +1,24 @@
+import Datepicker from 'react-tailwindcss-datepicker'
+import { useState } from 'react'
+
+const FilterDatepicker = () => {
+  const [value, setValue] = useState()
+  const handleChange = (value) => {
+    setValue(value)
+    console.log('Filtering contracts by: date range - ', value)
+  }
+
+  return (
+    <Datepicker
+      displayFormat='DD/MM/YYYY'
+      onChange={handleChange}
+      primaryColor='cyan'
+      value={value}
+      separator='to'
+      placeholder='Select period'
+      containerClassName='overflow-x-clip'
```

Comment:
So this is the bug I was talking about.
for the date picker to be able to pop up on top of everything, (outside of the boundaries of the sidebar) It needs to add this prop -> `containerClassName='overflow-x-clip` (I have tried different types of overflow all with the same output)
So, the calendar icon appears on the right side of the screen( marked in red) and all the areas to the right(marked in yellow) would be clickable for the date picker... if you click anywhere in that area the date picker would open, as today I don't know how to fix it, I agree from our conversation that we could open an issue to investigate this further.
and decide another way to display this date picker or not use it in the sidebar
![Screenshot 2024-12-18 112825](https://github.com/user-attachments/assets/2d5562ce-f422-4e10-b505-58c1fba72cd2)